docker/debian.Dockerfile

```diff
@@ -0,0 +1,115 @@
+# -------------------------------------
+#  PrestaShop FlashLight: Debian image
+# -------------------------------------
+ARG PS_VERSION
+ARG PHP_VERSION
+ARG PHP_FLAVOUR
+FROM php:${PHP_FLAVOUR} AS base-prestashop
+ARG PS_VERSION
+ENV PS_FOLDER=/var/www/html
+
+# Install base tools
+RUN apt-get update \
+  && DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -qqy \
+  bash less vim git tzdata zip unzip curl jq netcat-traditional ca-certificates \
+  lsb-release libgnutls30 gnupg libmcrypt4 libiconv-hook1 \
+  nginx libnginx-mod-http-headers-more-filter libnginx-mod-http-geoip \
+  libnginx-mod-http-geoip libnginx-mod-stream \
+  && apt-get clean
```

Comment:
Given that there will be other apt later on, it's better not to clean at this level.

Reply:
fixed